### PR TITLE
[docs]fix docs of cce cluster, node and addon

### PR DESCRIPTION
**Pull request by Jason-Zhang9309**
**What this PR does / why we need it**:

**Which issue this PR fixes**:
*(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*
fixes #xxx

**Special notes for your reviewer**:

**Release note**:

```release-note
fix docs of cce cluster, node and addon
```

## PR Checklist

* [ ] Tests added/passed.
* [x] Documentation updated.
* [ ] Schema updated.

## Acceptance Steps Performed

```
NONE
```

### CODE REVIEW

File: docs/data-sources/cce_cluster.md [hunk 1]
```diff
@@ -69,11 +69,15 @@ In addition to all arguments above, the following attributes are exported:
 
 * `enterprise_project_id` - The enterprise project id of the cce cluster.
 
-**endpoints**
+* `endpoints` - The access addresses of kube-apiserver in the cluster.
 
-* `internal` - The address accessed within the user's subnet.
+* `endpoints/url` - The URL of the cluster access address.
```
Comment (ShiChangkuo) [inline docs/data-sources/cce_cluster.md]: suggest to use the format of nested block:
```Structure is documented below.```

Comment (Jason-Zhang9309) [reply docs/data-sources/cce_cluster.md]: fixed

Comment (Jason-Zhang9309) [reply docs/data-sources/cce_cluster.md]: fixed

File: docs/resources/cce_addon.md [hunk 2]
```diff
@@ -2,6 +2,10 @@
 subcategory: "Cloud Container Engine (CCE)"
 ---
 
+# huaweicloud_cce_node
+
+Provides a CCE addon resource.
```
Comment (Lance52259) [inline docs/resources/cce_addon.md]: within HuaweiCloud

Comment (Jason-Zhang9309) [reply docs/resources/cce_addon.md]: fixed

**Pull request merged**